po/pl.po

```diff
@@ -186,18 +186,17 @@ msgid "The following kernels will be uninstalled"
 msgstr "Następujące jądra zostaną odinstalowane"
 
 #: src/Common/LinuxKernel.vala:1108 src/lib/l.misc.vala:12
-#, fuzzy
 msgid "Continue? (y/N): "
-msgstr "Kontynuować?"
+msgstr "Kontynuować? (y/N): "
 
 #: src/Common/LinuxKernel.vala:1122
 msgid "No updates"
 msgstr "Brak aktualizacji"
 
 #: src/Common/LinuxKernel.vala:1124
-#, fuzzy, c-format
+#, c-format
 msgid "Install Kernel Version %s ? (y/N): "
-msgstr "Zainstalować wersję jądra %s? (t/n): "
```

Comment:
Is this actually correct?
When the user gets this prompt, will pressing the "t" key on a Polish match getc() or read_line() looking for 'y' ?
The 'y' in the switch statement in l.misc ask() is a literal byte, not localized.
I'm not sure how to make that part localized yet.
Let me see if I can figure out a way to make the answer localized so that the prompt can be localized.

Reply:
Yes, "t" if localized.

Reply:
If you press the 't' key on a polish keyboard, it produces the same ascii byte value as 'y' in us-ascii? I somehow doubt this.
However, I think I have made ask() fully localized now.
Now the prompt and the answer from the user are both localized strings, instead of the answer being a raw byte.

Update your copy and look at the end of pl.po and now it also contains
```
#: src/lib/l.misc.vala:16
msgid "y"
msgstr ""

#: src/lib/l.misc.vala:17
msgid "n"
msgstr ""
```

You only have to supply the lower-case versions here because the users answer is converted to lower-case by string.down() before being examined.

Reply:
Also I apologize for mis-reading the initial commit. You were actually *removing* the (t/n) and putting in (y/N) which would have been correct since ask() was not localized at that time. Anyway, now that it is localized you can make the prmpt say "(t/N)" or whatever is correct for the N. Sorry for the confusion.